tests/optimization/test_lookup_finder.py

```diff
@@ -149,11 +161,33 @@ def get_user_assignment(self, obj, serializer_cls):
             return serializer_cls(obj.user_assignment[0]).data
         return None
 
-    @v.hints.from_types_of(get_lesson_title_list, "course")
+    @hints.from_types_of(get_lesson_title_list, "course")
     def get_lesson_titles(self, course, get_lesson_title_list_helper):
         return get_lesson_title_list_helper(course)
 
 
+class LessonSerializer(serializers.ModelSerializer):
+    facilitator_users = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Lesson
+        virtual_model = VirtualLesson
+        fields = [
+            "title",
+            "content",
+            "facilitator_users",
+        ]
+
+    @hints.defined_on_virtual_model()
```

Comment:
Why do we need the hint even though we have this field on the field list?

Reply:
Hints are always necessary if they're not a concrete model field. The idea is to be explicit on what should be found at the virtual model instead of the model.